app/controllers/PaypalController.scala

```diff
@@ -223,13 +167,19 @@ class PaypalController(ws: WSClient, paymentServices: PaymentServices, checkToke
       }
     }
   }
-  case class MetadataUpdate(marketingOptIn: Boolean)
 
-  val metadataUpdateForm: Form[MetadataUpdate] = Form(
-    mapping(
-      "marketingOptIn"->boolean
-    )(MetadataUpdate.apply)(MetadataUpdate.unapply)
-  )
+  implicit val UriWrites = new Writes[Uri] {
+    override def writes(uri: Uri): JsValue = JsString(uri.toString)
+  }
+
+  implicit val AuthResponseWrites = Json.writes[AuthResponse]
+
+  implicit val CountryGroupReads = new Reads[CountryGroup] {
```

Comment:
ignore this - I thought it was introduced in the pull request,  but actually it wasn't